src/Core/Domain/Shop/Exception/ShopAssociationNotFound.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
github syntax color does not apply to this file. any idea? wrong warning?

Reply:
Don't know it happens sometimes, but I don't know the reason why